app/views/administrate/application/_collection.html.erb

```diff
@@ -18,13 +18,13 @@ to display a collection of resources in an HTML table.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Table
 %>
 
-<table class="collection-data">
+<table class="collection-data" aria-labelledby="page-title">
```

Comment:
Ah, nevermind, the `aria-describedby` defines the help text, not the accessible name. 
![screen shot 2016-01-19 at 4 37 42 pm](https://cloud.githubusercontent.com/assets/108163/12432739/017f04fa-becb-11e5-88a8-a1b18b32217b.png)